test/utils.py

```diff
@@ -120,39 +119,54 @@ def cli_safe(self, cmd):
         return out
 
     def identify(self):
-        return self.cli_safe("identify")
+        return self.cli_safe(["identify"])
 
     def add_peer(self, node):
-        self.cli_safe("addpeer {} {} {}".format(node.ip, node.port, node.seed))
+        self.cli_safe([
+            "addpeer",
+            node.ip,
+            node.port,
+            node.seed,
+        ])
 
     def remove_peer(self, peer_id):
-        self.cli_safe("removepeer {}".format(peer_id))
+        self.cli_safe([
+            "removepeer",
+            peer_id,
+        ])
 
     def list_peer(self):
-        return self.cli_safe("listpeer")
+        return self.cli_safe(["listpeer"])
 
-    def list_topic_peer(self, topics):
-        return self.cli_safe("listtopic {}".format(' '.join(topics)))
+    def list_topic_peer(self, topics=[]):
+        return self.cli_safe(["listtopicpeer"] + topics)
 
     def subscribe_shard(self, shard_ids):
-        self.cli_safe("subshard {}".format(' '.join(map(str, shard_ids))))
+        self.cli_safe(["subshard"] + shard_ids)
 
     def unsubscribe_shard(self, shard_ids):
-        self.cli_safe("unsubshard {}".format(' '.join(map(str, shard_ids))))
+        self.cli_safe(["unsubshard"] + shard_ids)
 
     def get_subscribed_shard(self):
-        return self.cli_safe("getsubshard")
+        return self.cli_safe(["getsubshard"])
 
     def broadcast_collation(self, shard_id, num_collations, collation_size, collation_time):
-        self.cli_safe("broadcastcollation {} {} {} {}".format(
+        self.cli_safe([
+            "broadcastcollation",
             shard_id,
             num_collations,
             collation_size,
             collation_time,
-        ))
+        ])
+
+    def bootstrap(self, switch):
+        self.cli_safe([
+            "bootstrap",
+            "start" if switch else "stop",
```

Comment:
nitpick: `switch` here does not seem to function like a switch

Reply:
Do you have any suggestion on the name? I can only get `command` instead

Reply:
Used `mode` instead